lib/location_name_converter.rb

```diff
@@ -5,6 +5,10 @@ def self.to_internal_name(display_name)
     Location.where(display_name:).get(:name)
   end
 
+  def self.to_visible_internal_name(display_name)
```

Comment:
Since it's used in just one place, do we really need a separate helper instead of using it directly?

To be honest, I haven't liked the `LocationNameConverter` module since day one. Since we have the `Location` model ("We didn't have it when this module was introduced), we might deprecate it and use the `Location` model with its own helpers in the future. This isn't related to this PR.

Reply:
We definitely don't need a helper method for it.  I'll push a commit that inlines the behavior into the Clover helper method.